packages/vuetify/types/services/goto.d.ts

```diff
@@ -26,3 +26,5 @@ export interface GoToOptions {
   easing?: VuetifyGoToEasing
   appOffset?: boolean
 }
+
+export default function goTo(target: VuetifyGoToTarget, options?: GoToOptions): Promise<number>
```

Comment:
This has a different signature in types/index.d.ts

Reply:
No idea tbh, just updated the signature in both but I guess only one is used

Reply:
It was wrong signature there, fixed